### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by mattdesl** [pillars/lengthy#1]
Module looks great. I'm doing this:

```sh
npm i measures
node
$ m = require('measures')
```

And I'm getting:

```
Error: Cannot find module 'measures'
```

**Comment by karellm** [comment]
@mattdesl I should be clearer in the README that this module is not fully ready. Be aware that there might be some issues with it still and that it might change quite a bit in the future.

That said I just released `0.2.0` that should address your issue. I also updated the README. If you have questions or issues, please let me know, it will motivate me to get this finalized :)

**Comment by mattdesl** [comment]
Cool, gotcha. 😄
